recipe/meta.yaml

```
@@ -16,11 +16,14 @@ build:
   number: 1
   script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   script_env:
-    - MAX_JOBS=$CPU_COUNT
-    # Not compiling for 8.0;8.6;8.9;9.0+PTX to keep builds under 6 hours
-    - TORCH_CUDA_ARCH_LIST=8.0+PTX
+    # Limit MAX_JOBS in order to prevent runners from crashing
+    - MAX_JOBS=4
+    - TORCH_CUDA_ARCH_LIST=8.0;8.6;8.9;9.0+PTX
   skip: true  # [cuda_compiler_version in (undefined, "None")]
   skip: true  # [not linux]
+  # debugging skips below
+  skip: true  # [py!=313]
+  skip: true  # [cuda_compiler_version != "12.0"]
```

Comment out the lines below instead of removing so that they're there for future PRs when we need to debug things?

```suggestion
  # debugging skips below
  # skip: true  # [py!=313]
  # skip: true  # [cuda_compiler_version != "12.0"]
```